bucket/winlibs-mingw-llvm-ucrt.json

```diff
@@ -1,23 +1,23 @@
 {
     "homepage": "https://winlibs.com/",
     "description": "winlibs standalone build of GCC compiler and MinGW-w64 with LLVM and UCRT",
-    "version": "11.3.0-14.0.3-10.0.0-ucrt-r3",
+    "version": "12.1.0-14.0.4-10.0.0-ucrt-r2",
     "license": "GPL-2.0-only,BSD-2-Clause,Apache-2.0,ZPL-2.1",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/brechtsanders/winlibs_mingw/releases/download/11.3.0-14.0.3-10.0.0-ucrt-r3/winlibs-x86_64-posix-seh-gcc-11.3.0-llvm-14.0.3-mingw-w64ucrt-10.0.0-r3.7z",
-            "hash": "810a6a8361bc1a83bfe68745adbec30998a47b6a3f4d384505cffcc24a5dcd19",
+            "url": "https://github.com/brechtsanders/winlibs_mingw/releases/download/12.1.0-14.0.4-10.0.0-ucrt-r2/winlibs-x86_64-posix-seh-gcc-12.1.0-llvm-14.0.4-mingw-w64ucrt-10.0.0-r2.7z",
+            "hash": "0d9adaa44f9d812e31950b814500f266b6a0e1ef564660f48701da40b7860645",
             "extract_dir": "mingw64"
         },
         "32bit": {
-            "url": "https://github.com/brechtsanders/winlibs_mingw/releases/download/11.3.0-14.0.3-10.0.0-ucrt-r3/winlibs-i686-posix-dwarf-gcc-11.3.0-llvm-14.0.3-mingw-w64ucrt-10.0.0-r3.7z",
-            "hash": "129acf5a5696b751177f68342b0b02a1e906377307461ba894e2c9b8156a8f9a",
+            "url": "https://github.com/brechtsanders/winlibs_mingw/releases/download/12.1.0-14.0.4-10.0.0-ucrt-r2/winlibs-i686-posix-dwarf-gcc-12.1.0-llvm-14.0.4-mingw-w64ucrt-10.0.0-r2.7z",
+            "hash": "a4078f9e9038a81798ddd895d0a60f715780453c6f84bde8fde8f874204288b4",
             "extract_dir": "mingw32"
         }
     },
     "env_add_path": "bin",
     "checkver": {
-        "regex": "/download/(?<version>((?<gccVersion>11[\\d.]+)-(?<llvmVersion>[\\d.]+)-(?<mingwVersion>[\\d.]+)-ucrt-(?<revision>[\\w]+)))"
+        "regex": "/download/(?<version>((?<gccVersion>12[\\d.]+)-(?<llvmVersion>[\\d.]+)-(?<mingwVersion>[\\d.]+)-ucrt-(?<revision>[\\w]+)))"
```

Comment:
@PhyX-Meow Why hardcode the version here? Wouldn't it still work if the `12` was removed?

Reply:
https://github.com/chawyehsu/dorado/pull/609#issuecomment-1166753665

Reply:
Of course!

But perhaps `1[23]` would allow a major version bump, and then we'd then manually bump it the regex to `1[34]`?